submissions/2025-03-17-Curated-tmQM-xtb-Dataset-T=100K-Dataset-Restricted-to-Pd-Zn-Fe-Cu-v0.0/README.md

```diff
@@ -0,0 +1,56 @@
+# Curated tmQM-xtb Dataset: T=100K Dataset Restricted to Pd, Zn, Fe, Cu v0.0
+
+### Description
+
+This dataset was generated starting from the tmQM dataset (release 13Aug2024, https://github.com/uiocompcat/tmQM) containing 108541 unique molecules; each molecule was evaluated using gfn2-xtb, and then a short MD simulation performed to provide additional configurations of the molecules. Further details can be found in the hosting repository: https://zenodo.org/records/15015000. This dataset contains 23134 unique systems with 208141 total configurations.  This dataset is limited to systems that contain transition metals Pd, Zn, Fe, or Cu, and also only contain elements C, H, P, S, O, N, F, Cl, or Br with charges: {-1,0,+1}. Run with the BP86/def2-TZVP for singlepoint calculations at geometries from molecular dynamics using gfn-xtb, generating the properties: 'energy', 'gradient', 'dipole', 'quadrupole', 'wiberg_lowdin_indices', 'mayer_indices', 'lowdin_charges', 'dipole_polarizabilities', 'mulliken_charges'.
+
+
+### General Information
+
+- Date: 2025-03-17
+- Class: OpenFF Singlepoint Dataset
```

Comment:
I'm not sure we actually have this class in QCSubmit, it would be BasicDataset; although given that the results of this dataset can't actually be loaded into OpenFF I'd remove this field.